app/core/tests/test_commands.py

```diff
@@ -0,0 +1,23 @@
+from unittest.mock import patch
+
+from django.core.management import call_command
+from django.db.utils import OperationalError
+from django.test import TestCase
+
+
+class CommandTests(TestCase):
+
+    def test_wait_for_db_ready(self):
+        """Test waiting for db when db is available"""
+        with patch('django.db.utils.ConnectionHandler.__getitem__') as gi:
+            gi.return_value = True
+            call_command('wait_for_db')
+            self.assertEqual(gi.call_count, 1)
+
+    @patch('time.sleep', return_value=True)
```

Comment:
It's good practice to patch only the imported method. Not sure if with `time.sleep` this works, but it would be
```suggestion
    @patch('core.management.commands.wait_for_db.sleep', return_value=True)
```
if you had imported `from time import sleep` in the command file.

Though, fiddling with `time` in python testing can be cumbersome. But this great 3rd party lib helps and is what we use in our code base:
 https://github.com/spulec/freezegun

Reply:
Changed, and it still works as intended.